components/LessonCard.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+
+import '../scss/lessonCard.scss'
+
+type Props = {
+  coverImg: string
+  title: string
+  challengeCount: number
+  lessonCount: number
+  hourCount: string | number
+  description: string
+}
+
+const LessonCard: React.FC<Props> = props => {
+  return (
+    <div className="card shadow-sm mt-3">
+      <div className="card-body p-2">
+        <div className="row no-gutters">
+          <div className="col-2 mr-4">
+            <img
+              className="card-img h-100"
```

Comment:
do you need `card-img` and `h-100` class?

Reply:
Removed and added mw-100 to container instead. This was put in to avoid columns overlapping in small screen sizes. Now columns stack